### ISSUE TITLE
Error compiling

### ISSUE THREAD

**Issue by pmprog** [JonnyH/xcapoc-resources#1]
Thought I'd try and build this so I can make me a collection of all the graphics

```
/home/user/Code/xcapoc-resources/apoclib/city_tile.cpp: In member function ‘void ApocRes::CityMapTile::WriteXML(tinyxml2::XMLElement*)’:
/home/user/Code/xcapoc-resources/apoclib/city_tile.cpp:99:14: error: ‘class tinyxml2::XMLElement’ has no member named ‘SetText’
    voxelRow->SetText(ss.str().c_str());
              ^
make[2]: *** [CMakeFiles/apoclib.dir/apoclib/city_tile.cpp.o] Error 1
make[1]: *** [CMakeFiles/apoclib.dir/all] Error 2
```

**Comment by pmprog** [comment]
Okay, if I build the version of TinyXML in the OpenApoc/dependancies, it gets further, but then fails here

```
Linking CXX executable load_buildings
CMakeFiles/load_buildings.dir/load_buildings.cpp.o: In function `main':
load_buildings.cpp:(.text+0x3cb): undefined reference to `tinyxml2::XMLDocument::XMLDocument(bool, tinyxml2::Whitespace)'
load_buildings.cpp:(.text+0x5d1): undefined reference to `tinyxml2::XMLDocument::SaveFile(char const*, bool)'
libapoclib.a(city_tile.cpp.o): In function `ApocRes::CityMapTile::WriteXML(tinyxml2::XMLElement*)':
city_tile.cpp:(.text+0x8bb): undefined reference to `tinyxml2::XMLElement::SetText(char const*)'
collect2: error: ld returned 1 exit status
make[2]: *** [load_buildings] Error 1
make[1]: *** [CMakeFiles/load_buildings.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by JonnyH** [comment]
Looks like it's still linking against the 'old' tinyxml2 library - you should be able to override this with:

**Comment by JonnyH** [comment]
..an option that wasn't yet implemented :)

Hopefully fixes with 244c9f8 - with luck that should use the 'correct' pkgconfig path now, if not you should be able to manually edit the TINYXML2_LIBRARY/LIBPNG_LIBRARY variables (in ccmake / cmake-gui) to point to the /full/path/to/libwhatever.so if it still detects them wrong...

**Comment by pmprog** [comment]
Works a charm now :) Thanks
